app/settings.py

```diff
@@ -1,46 +1,60 @@
+import json
 import logging
 import os
 
 
 LOGGING_LEVEL = logging.getLevelName(os.getenv("LOGGING_LEVEL", "DEBUG"))
 LOGGING_FORMAT = "%(asctime)s.%(msecs)06dZ|%(levelname)s: sdx-downstream: %(message)s"
 
-APP_ROOT = os.path.dirname(os.path.abspath(__file__))
-APP_TMP = os.path.join(APP_ROOT, 'tmp')
 
-# Default to true, cast to boolean
-SDX_STORE_URL = os.getenv("SDX_STORE_URL", "http://sdx-store:5000")
-SDX_TRANSFORM_CS_URL = os.getenv("SDX_TRANSFORM_CS_URL", "http://sdx-transform-cs:5000")
-SDX_TRANSFORM_CORA_URL = os.getenv("SDX_TRANSFORM_CORA_URL", "http://sdx-transform-cora:5000")
-SDX_SEQUENCE_URL = os.getenv("SDX_SEQUENCE_URL", "http://sdx-sequence:5000")
+def parse_vcap_services():
+    vcap_services = os.getenv("VCAP_SERVICES")
+    parsed_vcap_services = json.loads(vcap_services)
+    rabbit_config = parsed_vcap_services.get('rabbitmq')
+    rabbit_url = rabbit_config[0].get('credentials').get('uri')
+    rabbit_url2 = rabbit_config[1].get('credentials').get('uri') if len(rabbit_config) > 1 else rabbit_url
+
+    ftp_config = parsed_vcap_services.get('ftp')
+    ftp_user = ftp_config[0].get('ftp_user')
+    ftp_pass = ftp_config[0].get('ftp_pass')
+
+    return rabbit_url, rabbit_url2, ftp_user, ftp_pass
+
+if os.getenv("CF_DEPLOYMENT", False):
+    RABBIT_URL, RABBIT_URL2, FTP_USER, FTP_PASS = parse_vcap_services()
+else:
+    # Default to true, cast to boolean
+    SDX_STORE_URL = os.getenv("SDX_STORE_URL", "http://sdx-store:5000")
+    SDX_TRANSFORM_CS_URL = os.getenv("SDX_TRANSFORM_CS_URL", "http://sdx-transform-cs:5000")
+    SDX_TRANSFORM_CORA_URL = os.getenv("SDX_TRANSFORM_CORA_URL", "http://sdx-transform-cora:5000")
+    SDX_SEQUENCE_URL = os.getenv("SDX_SEQUENCE_URL", "http://sdx-sequence:5000")
+
+    FTP_HOST = os.getenv('FTP_HOST', 'pure-ftpd')
```

Comment:
Where does the FTP host come from if it's a CF deployment?  Is FTP an available CF service for us to use?